db/collection.go

```diff
@@ -473,14 +541,28 @@ func (t *Txn) createSaveActions(identity thread.PubKey, updated ...[]byte) ([]co
 			return nil, err
 		}
 
+		// Update readonly/protected mod tag
+		_, next = setModifiedTag(next)
+
+		// Because this is a save event, even though we might still create the new instance
+		// it has to have a valid _id ahead of time.
 		id, err := getInstanceID(next)
 		if err != nil {
 			return nil, err
 		}
 		key := baseKey.ChildString(t.collection.name).ChildString(id.String())
 		previous, err := t.collection.db.datastore.Get(key)
 		if err == ds.ErrNotFound {
-			return nil, errCantSaveNonExistentInstance
+			a := core.Action{
```

Comment:
This means we can "Save" an instance even if we haven't seem it before, so long as it already has a valid _id.

Reply:
This is useful for reasons I've outlined elsewhere (see PR description), but additionally, it circumvents a restriction on current Hub. With that in mind, we should _think through_ the implications of this a little bit. Seems safe on the outset.